docs/Release_Notes.txt

```diff
@@ -0,0 +1,497 @@
+M-modified; A-added; D-deleted
+
+What changes were made to the above code/scripts to support the new architecture?
+ The obsproc software has been consolidated into its own network. As such, the directories are bigger. 
+ However, because code is under one umbrella, the PBS trigger / ecflow scripts and J-job scripts are more streamlined.
+ The obsproc software has been moved to github and now utilizes cmake to build and install the package. 
+ https://github.com/NOAA-EMC/obsproc and build obsproc repo with cmake
+ The obsproc software no longer includes bufr_dump. 
+ bufr_dump has been moved to its own github repository and is installed as its own module. 
+ Separate bufr_dump.v1.0.0 Release Notes. See https://github.com/NOAA-EMC/bufr-dump
+
+Were any other changes made that aren’t directly related to the transition? 
+ No
+
+Are there any changes to incoming data needs or outgoing products? 
+ Network bufr dump and prepbufr files are now written to obsproc’s com, not each network’s com. See notes in the next section.
+
+If output filenames are changing, list the current and new filename. 
+ Output file names are not changing; however, their destination directory ($COMOUT) is changing. 
+ Network data produced by obsproc processing will be located in $COMROOT/obsproc/v1.0/${RUN}.YYYYMMDD, 
+ where ${RUN} is entities such as cdas, gfs, gdas, nam, rap, rtma, urma, dump (for dump_monitor).
+
+Compute resource information, for every job 
+ See triggers/ directory
+ Total tasks
+ Tasks per node
+ Serial or parallel or mpmd
+ Memory requirements (exclusive or shared)
+
+List any other PBS resource information requirements or MPI tunables
+ *** Providing PBS and/or ecFlow submission scripts as part of release tag is preferred; 
+ if they are provided then resource information is not needed in the release notes.
+
+Runtime changes compared to current production (/gpfs/dell1/nco/ops/com/logs/runtime/daily/* for current stats)
+ Optimization was not performed prior to release because of the team reshuffling and resultant tight deadline. 
+ Obsproc developers will work with assigned SPA to meet optimization goals.
+ JOBSPROC_CDAS_PREP2* was not fully tested.This processing requires more days than were available in /canned com. 
+ We also could not use live dcom data once it became available since this processing requires sice.YYYYMMDD data files, which are not yet available in realtime.
+
+Disk space required per day or per cycle; data retention on disk will remain the same unless otherwise requested.
+ Same as WCOSS1
+
+Dissemination will remain the same unless otherwise communicated to NCO and a PNS/SCN issued
+
+HPSS archive retention will remain the same unless approval granted by HPCRAC
+
+What are your up and downstream dependencies?
+ Upstream: decoders, satingest ($DCOMROOT)
+ Downstream: gfs and gdas model and analysis runs; omb sice data
+
+Provide a pointer to your COMOUT, temp directory and pbspro output file that was used during testing:
```

Comment:
I am not sure I understand the point of this in a Release Note.

Reply:
Reply to this and the earlier comment - these were the questions posed by NCO, we answered them. I can't answer for past decisions before my time.